src/App.ts

```diff
@@ -448,17 +454,17 @@ export default class App {
       );
     } else if (authorize === undefined && !usingOauth) {
       throw new AppInitializationError(
-        `No token, no authorize, and no oauth installer options provided. ${tokenUsage}`,
+        `${tokenUsage} \n\nSince you have not provided a token or authorize, you might be missing one or more required oauth installer option(s). See https://slack.dev/bolt-js/concepts#authenticating-oauth for these required fields.\n`,
```

Comment:
Typically we get to this when the user is missing some oauth installer options. This specific error is highly confusing since it's not necessarily true that "no oauth installer options provided".

Reply:
This is definitely a step forward is helping the developer troubleshoot the error. Nice work!